edn/tests/tests.rs

```diff
@@ -17,9 +17,22 @@ use std::iter::FromIterator;
 use num::bigint::ToBigInt;
 use num::traits::{Zero, One};
 use ordered_float::OrderedFloat;
+use edn::keyword;
+use edn::types::Value;
 use edn::types::Value::*;
 use edn::parse::*;
 
+// Helper for making wrapped keywords with a namespace.
+fn k_ns(ns: &str, name: &str) -> Value {
```

Comment:
Let's spend some time to learn `Into` and `From`; these are solved problems in Rust-land.